CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Breaking changes
+- Due to a critical vulnerability with CVSS score of 10.0 that was discovered
+affecting all `Salt Master` versions inferior to `3000.1`, this release ships
+with a hot fix and of course a `Salt Master` version updated to `3000.3`.
+Users, especially those who expose the `Salt Master` to the internet must
+therefore upgrade immediately.
+
+- The official `python-kubernetes client v11` has recently been released. The new
+client comes with new API changes and a couple of old APIs deprecated.
+MetalK8s has been updated to fully support this new client API version.
+
+### Features added
+- [#650](https://github.com/scality/metalk8s/issues/650) - Upgrade Salt master
+to version `3000.3`
+(PR [#2549](https://github.com/scality/metalk8s/pull/2549))
+
+- [#2029](https://github.com/scality/metalk8s/issues/2029) - Support
+python-kubernetes client `v11`
+(PR [#2554](https://github.com/scality/metalk8s/pull/2554))
+
+
+### Enhancements
+- Add and allow `etcd` state to run health check by default
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- Add pillar refresh to ensure deploy node is resilient
+(PR [#2147](https://github.com/scality/metalk8s/pull/2147))
+
+- [#2530](https://github.com/scality/metalk8s/issues/2530) - Fix cluster
+upgrade with deployed workloads failures due to disallowed disruptions
+(PR [#2531](https://github.com/scality/metalk8s/pull/2531))
+
+- Fix apiserver-proxy upstreams
+(PR [#2555](https://github.com/scality/metalk8s/pull/2555))
+
+- Add documentation on how to downgrade Salt if needed
+(PR [#2540](https://github.com/scality/metalk8s/pull/2540))
```

Comment:
This is important, and should go in "Upgrade notes" or "Breaking changes": once upgrade to 2.4.4 is done, downgrade can only be achieved manually (and the documentation to do it has been added by the PR you link here).